src/main/js/store/actions/types.js

```diff
@@ -25,6 +25,11 @@ export const PROJECTS_FETCHING = 'PROJECTS_FETCHING';
 export const PROJECTS_FETCHED = 'PROJECTS_FETCHED';
 export const PROJECTS_FETCH_ERROR = 'PROJECTS_FETCH_ERROR';
 
+// User email Actions
+export const USER_EMAIL_FETCHING = 'USER_EMAIL_FETCHING';
+export const USER_EMAIL_FETCHED = 'USER_EMAIL_FETCHED';
+export const USER_EMAIL_FETCH_ERROR = 'USER_EMAIL_FETCH_ERROR';
```

Comment:
Are these still neccessary? It doesn't look like anything here is part of redux.